source/tutorials/feature-classifier.rst

```diff
@@ -81,7 +81,7 @@ Finally, we verify that the classifier works by classifying the representative s
 
 .. command-block::
 
-   qiime feature-classifier classify \
+   qiime feature-classifier classify-sklearn \
```

Comment:
Can you also make this fix in the Moving Pictures tutorial? I think that's the only other place needing updates.